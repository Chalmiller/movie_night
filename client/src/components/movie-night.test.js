import React from 'react';
import { shallow } from 'enzyme';
import MovieNight from './movie-night.component';

it('renders movie night without crashing', () => {
  expect(shallow(<MovieNight/>)).toMatchSnapshot();
});