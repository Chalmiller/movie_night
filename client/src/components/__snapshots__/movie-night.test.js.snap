// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders movie night without crashing 1`] = `ShallowWrapper {}`;
